repository_service_tuf_worker/repository.py

```diff
@@ -1022,8 +1079,24 @@ def _root_metadata_update(
         """
         current_root: Metadata[Root] = self._storage_backend.get(Root.type)
 
-        self._trusted_root_update(current_root, new_root)
+        # Do not update the Root if still not fully signed
+        signed = self._validate_signatures(new_root, Root.type)
```

Comment:
Also, there's some redundancy, if you're coming here from `_sign_root`, where you just called the same _validate_signatures with the same argument. Probably no big deal,  but still obsolete cycles.

Reply:
I'm limiting the PR scope to enable the Distributed Asynchronous Signing (DAS) to the bootstrap only.
It will limit the implementation complexity by user story, making even small changes.
You can see in [Feature: support distributed async threshold signing.](https://github.com/repository-service-tuf/repository-service-tuf/issues/327)